packages/app/components/claim/claim-form.tsx

```diff
@@ -124,11 +125,11 @@ export const ClaimForm = ({ edition }: { edition: CreatorEditionResponse }) => {
     );
   }
 
-  if (state.status === "loading" && state.signaturePrompt === false) {
-    router.pop();
+  // if (state.status === "loading" && state.signaturePrompt === false) {
```

Comment:
I'm not sure if need this, if I open it will not be able to enter this screen. 
or should disable the `Claim for Free` button instead of this? any thought? 
cc @axeldelafosse @intergalacticspacehighway

Reply:
yeah, we should disable it for now and show a text maybe `claim in progress`?

Reply:
looks good, will do it.